Sources/StreamChat/APIClient/Endpoints/Payloads/MessageReactionPayload.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 
 /// The type describes the incoming message-reaction JSON.
 struct MessageReactionPayload: Decodable {
-    private enum CodingKeys: String, CodingKey, CaseIterable {
+    internal enum CodingKeys: String, CodingKey, CaseIterable {
```

Comment:
That's a good one to be discussed. I would either always explicitly specify the `internal` access level or always skip it. I'm OK with both as long as we're have consistency
```suggestion
    enum CodingKeys: String, CodingKey, CaseIterable {
```